playwright-e2e/pages/pancan/home/home-page.ts

```diff
@@ -10,25 +12,26 @@ export default class HomePage extends PancanPage implements HomePageInterface {
   async waitForReady(): Promise<void> {
     await expect(this.getLogInButton()).toBeVisible();
     await expect(this.getJoinCountMeInButton()).toBeVisible();
+
   }
 
   getLogInButton(): Locator {
-    return this.page.locator('.header button[data-ddp-test="signInButton"]:has-text("Log In")');
+    return this.page.locator('.header .controls button[data-ddp-test="signInButton"]:has-text("Log In")');
```

Comment:
I believe `data-ddp-test` attri alone in locator is sufficient. Rest is not needed.

Reply:
it seems to have two matches with that locator. That's why I had to be more specific